src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ballNumberOne = new Lottery().getRandomBall();
+        Ball ballNumberTwo = new Lottery().getRandomBall();
+        Ball ballNumberThree = new Lottery().getRandomBall();
+        System.out.println(ballNumberOne.toString());
+        System.out.println(ballNumberTwo.toString());
+        System.out.println(ballNumberThree.toString());
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/193417144-fbd465d9-f40e-4304-a30e-415ac6c6324d.png)
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected